lxd/api.go

```diff
@@ -106,7 +106,8 @@ func restServer(d *Daemon) *http.Server {
 		uiHandlerErrorUINotEnabled := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 			w.Header().Set("Content-Type", "text/html")
 			w.WriteHeader(http.StatusServiceUnavailable)
-			fmt.Fprint(w, errorMessage)
+			_, err := fmt.Fprint(w, errorMessage)
+			logger.Warn("Could not write error message to writer object", logger.Ctx{"url": r.URL, "method": r.Method, "remote": r.RemoteAddr, "err": err})
```

Comment:
I think

> Failed sending error message to client

Would be clearer?

Reply:
Although, as mentioned earlier in this PR, I'm not sure we actually want to be considering write errors for metrics or logging.

Reply:
I think keeping this logging makes sense to make the failure visible in the logs as well, even if the falty part is not the server. Also, this change is included here because of the linter mostly.

I will update the log message as you suggested.

Reply:
> I think keeping this logging makes sense to make the failure visible in the logs as well, even if the falty part is not necessarily the server. Also, this change is included here because of the linter mostly.

OK lets see if it introduces any log noise and we can reevaluate